cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -391,10 +391,7 @@ def test_no_template_power_control(provider, soft_assert):
     """
     view = navigate_to(provider, 'ProviderTemplates')
     view.toolbar.view_selector.select('Grid View')
-    if provider.appliance.version < '5.10':
-        soft_assert(not view.toolbar.power.is_enabled, "Power enabled in template grid view!")
-    else:
-        soft_assert(not view.toolbar.power.is_displayed, "Power displayed in template grid view!")
+    soft_assert(not view.toolbar.power.is_displayed, "Power displayed in template grid view!")
 
```

Comment:
@lina-nikiforova 
According to me, as we are asserting for once in this test case. So it would be better approach if we use ordinary ```assert``` instead of ```soft_assert```

Reply:
@ganeshhubale  there're 2 more soft asserts in this test, e.g. https://github.com/ManageIQ/integration_tests/pull/8344/files#diff-cbdf78293430cdea45837cb22c5774a2R406

Reply:
My bad! Thanks for explaination. :)